docs/user-guide/config-management-plugins.md

```diff
@@ -450,17 +440,16 @@ spec:
     The `lockRepo` key is not relevant for sidecar plugins, because sidecar plugins do not share a single source repo
     directory when generating manifests.
 
-### 2. Write discovery rules for your plugin
+### 2. Write discovery rules for your plugin, or use the plugin name
```

Comment:
As noted above, don't include numbers or periods or colons in section headings in general as they're really painful for link / search purposes.

Use very short text items so that users can scan/click/etc.